frontend/epfl-memento/templates/includes/card-info.inc.php

```diff
@@ -1,45 +1,65 @@
 <?php
 namespace EPFL\Plugins\Gutenberg\Memento;
 
-$start_date = date_format(date_create($event->start_date), 'd-m-Y');
-$end_date   = date_format(date_create($event->end_date), 'd-m-Y');
-$start_time = date_format(date_create($event->start_time), 'H:i');
-$end_time   = date_format(date_create($event->end_time), 'H:i' );
+$start_date = date_format(date_create($event->start_date), 'd.m.Y');
+$end_date   = date_format(date_create($event->end_date), 'd.m.Y');
+$start_time = !empty($event->start_time) ? date_format(date_create($event->start_time), 'H:i') : '';
+$end_time   = !empty($event->end_time)   ? date_format(date_create($event->end_time), 'H:i')   : '';
 
 $datastart_date = date_format(date_create($event->start_date), 'Y-m-d') . 'T' . $start_time;
 $dataend_date = date_format(date_create($event->end_date), 'Y-m-d') . 'T' . $end_time;
 
-$markup .= '<span class="card-info-date" itemprop="startDate" content="'. $datastart_date .'">' . esc_html($start_date) . '</span>';
 
-if ($event->start_date == $event->end_date && !empty($startime) && !empty($endtime)){
-    $markup .= '<span>' . esc_html($start_time) . '</span>';
-    $markup .= '<span>' . esc_html($end_time) . '</span>';
+if ($event->start_date == $event->end_date){
+    $markup .= '<svg class="icon feather" aria-hidden="true"><use xlink:href="#calendar"></use></svg> ';
+    $markup .= '<span class="card-info-date event-date" itemprop="startDate" content="'. $datastart_date .'">' . esc_html($start_date) . '</span>   &nbsp;';
```

Comment:
Sometimes the concatenation is '`␣.␣$var␣.␣`' and sometimes it's '`.␣$var␣.`', it should rather be always the same.